packages/react-dom/src/__tests__/ReactEmptyComponent-test.js

```diff
@@ -136,6 +136,8 @@ describe('ReactEmptyComponent', () => {
   });
 
   it('should distinguish between a script placeholder and an actual script tag', () => {
+    const consoleSpy = spyOnDev(console, 'error');
```

Comment:
For warnings, we use toWarnDev() matcher. Could you please use that?

Reply:
I have a problem with this as I already use `expect` to check that there is no error. I tried to google if I can chain expects in jest, but I didn't find how :/.

Reply:
>I already use expect to check that there is no error.

I'm not sure what this means, but if you mean `not.toThrow()`, you can just remove that completely. It doesn't mean anything. If it threw, the test would fail anyway.